packages/ng/tag/tag.component.html

```diff
@@ -1,11 +1,12 @@
 @if (link) {
 <a [routerLink]="link" class="tag" [ngClass]="tagClasses" [class.mod-outlined]="outlined">
 	@if (icon) {
-	<lu-icon [icon]="icon"></lu-icon><span class="tag-content">{{ label }}</span> } @else { {{ label }} }</a
+	<lu-icon [icon]="icon"></lu-icon><span class="tag-content">{{ label }}</span> } @else { <span class="tag-content">{{ label }}</span> }</a
```

Comment:
Je pense qu'on peut désormais simplifier la condition en supprimant le `@else` :

```
@if (icon) {
  <lu-icon [icon]="icon"></lu-icon>
}
<span class="tag-content">{{ label }}</span>
```

Idem plus bas.

Reply:
You're absolutely right!